modules/nodes-edge-remote-workers-strategies.adoc

```diff
@@ -148,13 +151,12 @@ tolerations:
 - key: "node.kubernetes.io/not-ready"
   operator: "Exists"
   effect: "NoExecute" <2>
-  tolerationSeconds: 600
+  tolerationSeconds: 600 <3>
 ...
 ----
 <1> The `NoExecute` effect without `tolerationSeconds` lets pods remain forever if the control plane cannot reach the node.
 <2> The `NoExecute` effect with `tolerationSeconds`: 600 lets pods remain for 10 minutes if the control plane marks the node as `Unhealthy`.
-
-{product-title} uses the `tolerationSeconds` value after the `pod-eviction-timeout` value elapses.
+<3> You can specify your own `tolerationSeconds` value.
 
 Other types of {product-title} objects::
```

Comment:
```suggestion

Other types of {product-title} objects::
```
This is previewing as part of the footnote, but it looks like it ought to be a glossary-style section.

Reply:
I think I fixed it, wdyt?

Reply:
Looks good to me!